.github/workflows/validate-binaries.yml

```diff
@@ -45,5 +45,7 @@ jobs:
       repository: "pytorch/ao"
       with_cuda: "enable"
       with_rocm: "disable"
-      smoke_test: "source ./.github/scripts/validate_binaries.sh"
+      smoke_test: |
+          pip install -r dev-requirements.txt
```

Comment:
best to remove this line and only install torch, if the binaries crash because of a missing dependency that's bad

Reply:
My hope was that we can run the full test suite on the built binaries, and our test require dev-reqs to run

Reply:
Ok maybe before installing requirements then you can run python -c "import torch"